apps/etterlatte-behandling/src/main/kotlin/behandling/generellbehandling/GenerellBehandlingDao.kt

```diff
@@ -0,0 +1,71 @@
+package no.nav.etterlatte.behandling.generellbehandling
+
+import com.fasterxml.jackson.module.kotlin.readValue
+import no.nav.etterlatte.behandling.hendelse.getUUID
+import no.nav.etterlatte.libs.common.objectMapper
+import no.nav.etterlatte.libs.database.setJsonb
+import no.nav.etterlatte.libs.database.singleOrNull
+import no.nav.etterlatte.libs.database.toList
+import java.sql.Connection
+import java.sql.ResultSet
+import java.util.*
+
+class GenerellBehandlingDao(private val connection: () -> Connection) {
+    fun opprettGenerellbehandling(generellBehandling: GenerellBehandling) {
+        with(connection()) {
+            val statement =
+                prepareStatement(
+                    """
+                    INSERT INTO generellbehandling(id, type, innhold, sak_id)
+                    VALUES(?::UUID, ?, ?, ?)
```

Comment:
```suggestion
                    VALUES(?, ?, ?, ?)
```
cast'en er vel ikke nødvending

Reply:
50/50 tenker at det ikke kan skade, tenker det er greit å være eksplisitt mtp at vi har sendt inn string her flere ganger tidligere.